bokehjs/src/coffee/models/widgets/main.coffee

```diff
@@ -1,34 +1,36 @@
 module.exports = {
-  editors:                  [require('./cell_editors'), 'Editor']
-  formatters:               [require('./cell_formatters'), 'Formatter']
+  models: {
```

Comment:
Also we need to note that `bokeh-api.js` has to be included explicitly now, in any case.